hwdb/60-sensor.hwdb

```diff
@@ -57,6 +57,11 @@
 # automatically flip their output for an upside-down display when the device
 # is held upright.
 #
+#    ACCEL_LOCATION=<location>
+#
+# where <location> is the location of the sensor. This value could be 'base'
+# or 'display'. The default, when unset, is equivalent to:
```

Comment:
It looks like you forgot to add some text here after "The default, when unset, is equivalent to:" I guess that should read: "The default, when unset, is equivalent to: 'display'"

Reply:
Thanks. It's fixed now